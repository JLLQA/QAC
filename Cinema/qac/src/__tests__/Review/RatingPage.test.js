import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';

test('It matches snapshot', () => {
    const tree = renderer.create
    expect(tree).toMatchSnapshot();
})