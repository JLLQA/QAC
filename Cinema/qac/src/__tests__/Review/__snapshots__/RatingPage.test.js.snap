// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches snapshot 1`] = `[Function]`;

exports[`Testing Rating page functionality It matches snapshot 1`] = `[Function]`;
